sdk/src/Services/DynamoDBv2/Custom/DataModel/Configs.cs

```diff
@@ -152,12 +162,6 @@ public DynamoDBContextConfig()
 #endif
     public class DynamoDBOperationConfig : DynamoDBContextConfig
```

Comment:
Should we break this hierarchy while we're here?

We're discouraging the use of `DynamoDBOperationConfig` in V4 via `[Obsolete]` #3421, but this did lead to the confusion that this is addressing in the first place.

Reply:
I think it make sense to break the hierarchy. I assume we would copy over the properties from DynamoDBContextConfig to DynamoDBOperationConfig that made sense at the operation level.

Reply:
Done in https://github.com/aws/aws-sdk-net/pull/3422/commits/f478a1b33f0b6c2c4c12e525aaf0aaf25318d2c5